### PR TITLE
Improve Print Layout

### PR DESCRIPTION
**Pull request by nitro-marky**
## What does this change?
This makes the print layout a little more acceptable by removing certain page elements. (There is potentially more to do but this is a start). There are still edge cases, like atoms-rendering but this is a start.

### Before
Screenshot 2020-12-14 at 11 30 30

### After
Screenshot 2020-12-14 at 11 30 11

### CODE REVIEW

File: src/web/layouts/StandardLayout.tsx [hunk 1]
```diff
@@ -336,22 +336,26 @@ export const StandardLayout = ({
     const { branding } = CAPI.commercialProperties[CAPI.editionId];
     return (
         <>
-            <div>
+            <div data-print-layout='hide'>
```
Comment (liywjl) [inline src/web/layouts/StandardLayout.tsx:339]: do you need to repeat `data-print-layout='hide'` on child DOM nodes?

File: src/web/layouts/StandardLayout.tsx [hunk 2]
```diff
@@ -559,24 +568,25 @@ export const StandardLayout = ({
             </Section>
 
             <Section
+                data-print-layout='hide'
```
Comment (liywjl) [inline src/web/layouts/StandardLayout.tsx:571]: do you need to repeat data-print-layout='hide' on child DOM nodes?

**Review by liywjl — CHANGES_REQUESTED** [review]
I do not think you need to repeat `data-print-layout='hide'` on child DOM nodes

File: src/web/layouts/StandardLayout.tsx (hunk 2 above)
Comment (nitro-marky) [reply src/web/layouts/StandardLayout.tsx:563]: I wasn't sure as it was outside of a div, I'll give it a try. I've already removed some of the nesting

File: src/web/layouts/StandardLayout.tsx (hunk 1 above)
Comment (nitro-marky) [reply src/web/layouts/StandardLayout.tsx:339]: Where does standard layout get that attribute from?

Comment (nitro-marky) [reply src/web/layouts/StandardLayout.tsx:339]: Where would you recommend putting that attribute?

**Comment by nitro-marky** [comment]
[Print Layout - Hide elements](https://trello.com/c/HeNfqVlw/1778-print-layout-hide-elements)

File: src/web/layouts/StandardLayout.tsx (hunk 1 above)
Comment (nitro-marky) [reply src/web/layouts/StandardLayout.tsx:339]: I tried changing the attribute up the hierarchy and it failed to work. Unless there are other methods I'm happy that this works for now.

**Review by liywjl — APPROVED** [review]
👍

**Pull request merged**